bin/hash_file

```diff
@@ -7,11 +7,11 @@ fi
 
 # `shasum` is available on only macOS
 if command -v shasum &> /dev/null; then
-  sha_command='shasum -a 256'
+  sha_command=(shasum -a 256)
 else
-  sha_command='sha256sum'
+  sha_command=(sha256sum)
 fi
 
 # Both `shasum` and `sha256sum` will print the hash and the file name (`$1`).
 # We only care about the hash, so we use `cut` to extract it.
-$sha_command "$1" | cut -f1 -d " "
+"${sha_command[@]}" "$1" | cut -f1 -d " "
```

Comment:
This was updated to use a bash Array and then `"${array[@]}"` quoted array expansion syntax, to make sure we handle the case of spaces properly, and especially fix the `shellcheck` violation `SC2086` we had in https://buildkite.com/automattic/ci-toolkit-buildkite-plugin/builds/625#018fc58c-b850-4838-bc26-8cf8868c8289/163-174